doc/changelog.rst

```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+.. rubric:: 0.2.0
+
+- Remove hard dependency on cupy Python package, so that users can install a
+  binary wheel for it.
+- Require at least cupy 13.4 to avoid a bug with non-blocking transfers.
+- Change all iterators to be asynchronous, to support future integration with
+  data that arrives from an asynchronous source such as the network.
+- Correct documentation to indicate that Python 3.12 is the minimum supported
+  version.
```

Comment:
I was about to say "I don't remember reviewing this" but I see it was done a while back in #16 .